docs/ref/django-admin.txt

```diff
@@ -1299,7 +1299,7 @@ don't.
 
     This is a known limitation. It arises from the need to serialize objects
     in order to exchange them between processes. See
-    :ref:`python:pickle-picklable` for details.
+    :ref:`python:pickle-pickleable` for details.
```

Comment:
Similarly, can I change:
```
django/test/runner.py:    def check_pickleable(self, test, err):
django/test/runner.py:        # Ensure that sys.exc_info() tuples are pickleable. This displays a
django/test/runner.py:        self.check_pickleable(test, err)
django/test/runner.py:        self.check_pickleable(test, err)
django/test/runner.py:        self.check_pickleable(test, err)
django/test/runner.py:        - make tracebacks pickleable with tblib, if available
django/forms/widgets.py:# Defined at module level so that CheckboxInput is pickleable (#17976)
docs/topics/http/sessions.txt:  pickleable Python object when using
docs/ref/django-admin.txt:    :ref:`python:pickle-pickleable` for details.
docs/conf.py:# that aren't pickleable (module imports are okay, they're removed automatically).
docs/spelling_wordlist:pickleable
tests/forms_tests/tests/test_fields.py:    def test_boolean_pickleable(self):
tests/template_tests/test_response.py:        # known to be unpickleable (e.g., a function).
tests/template_tests/test_response.py:        # known to be unpickleable (e.g., a function).
tests/queries/models.py:        # is unpickleable doesn't stop you from cloning queries
tests/queryset_pickle/tests.py:        Test that a model not defined on module level is pickleable.
tests/validation/test_picklable.py:    def test_validationerror_is_pickleable(self):
```